docs/conf.py

```diff
@@ -0,0 +1,53 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+# -- Path setup --------------------------------------------------------------
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+# -- Project information -----------------------------------------------------
+
+project = "platformdirs"
+copyright = "2021, The platformdirs team"
+author = "The platformdirs team"
+
+# The full version, including alpha/beta/rc tags
+release = "2.0.0"
```

Comment:
This should be imported from the package not declared again.

Reply:
Oh and yes, definitely add it to the CI.

Reply:
I'm not familiar enough with Microsoft's CI system to figure out how the setup works and where to add it. Do you mind addressing that separately?